lib/features/copy-paste/BpmnCopyPaste.js

```diff
@@ -59,9 +59,17 @@ export default function BpmnCopyPaste(bpmnFactory, eventBus, moddleCopy) {
     }
   });
 
-  var references;
+  var referencesKey = '__bpmn-js-refs';
```

Comment:
Using the ID `__bpmn-js-refs` will break copy and paste. I know this is an edge case but this is why the cache wasn't used to store the references.

Example diagram:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_152mxu0" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.2.0" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.0.0">
  <bpmn:process id="Process_11wdnxl" isExecutable="true">
    <bpmn:startEvent id="__bpmn-js-refs" name="__bpmn-js-refs">
      <bpmn:messageEventDefinition id="MessageEventDefinition_1sfp0d2" messageRef="Message_2dgisck" />
    </bpmn:startEvent>
  </bpmn:process>
  <bpmn:message id="Message_2dgisck" name="Message_1" />
  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process_11wdnxl">
      <bpmndi:BPMNShape id="Event_10uizli_di" bpmnElement="__bpmn-js-refs">
        <dc:Bounds x="179" y="79" width="36" height="36" />
        <bpmndi:BPMNLabel>
          <dc:Bounds x="160" y="122" width="74" height="14" />
        </bpmndi:BPMNLabel>
      </bpmndi:BPMNShape>
    </bpmndi:BPMNPlane>
  </bpmndi:BPMNDiagram>
</bpmn:definitions>
```

![brave_bZsXtF6dPF](https://user-images.githubusercontent.com/7633572/184831319-5dbc954a-842c-49d4-ac9c-eb6c3a15365a.gif)

Reply:
Let's choose an identifier that starts with a non-ID prefix then, i.e. a number :wink:

Reply:
Or this: https://www.compart.com/en/unicode/U+1F697

Reply:
Now using a non-QName key to do the job: https://github.com/bpmn-io/bpmn-js/pull/1707/commits/7914e5da73f47b51951876c074e149edb4b90793#diff-6ec8b9f14d0e1b571840b0b87229ca40f6974c253b87200c33d53f62a4eeaaddR62.

Reply:
🤯